src/web-worker/amp-worker.js

```diff
@@ -76,7 +76,7 @@ class AmpWorker {
     // Use RTV to make sure we fetch prod/canary/experiment correctly.
     const useRtvVersion = !getMode().localDev && !getMode().test;
     const url = calculateEntryPointScriptUrl(
-        loc, 'ww', getMode().localDev, useRtvVersion);
+        loc, 'ww', !useRtvVersion, useRtvVersion, getMode().minified);
```

Comment:
Not sure about this change here. Without it, we're not pulling the web worker from local code. @choumx WDYT?